onnxruntime/core/providers/nnapi/nnapi_builtin/builders/shaper.h

```diff
@@ -95,58 +31,6 @@ class Shaper {
   void Clear();
 
```

Comment:
Can we get rid of this? 

It's generally simpler to create a new instance of the class if instead of having a 'clear' or 'reset' method that has to be kept in sync with all the member variables.